MapboxNavigation/BottomBannerView.swift

```diff
@@ -77,6 +101,34 @@ open class BottomBannerView: UIView {
         arrivalTimeLabel.text = "10:09"
     }
     
+    @objc public func navigationService(_ service: NavigationService, didRerouteAlong route: Route, at location: CLLocation?, proactive: Bool) {
+        refreshETA()
+    }
+    
+    @objc public func navigationService(_ service: NavigationService, didUpdate progress: RouteProgress, with location: CLLocation, rawLocation: CLLocation) {
+        resetETATimer()
+        updateETA(routeProgress: progress)
+        previousProgress = progress
+    }
+    
+    @objc func removeTimer() {
+        timer?.disarm()
+        timer = nil
+    }
+    
+    @objc func resetETATimer() {
```

Comment:
What's the reasoning behind resetting this timer every second?

Reply:
Good question. Logic was taken wholesale from the RMVC.

Reply:
Posterity: After discussions with @frederoni and @akitchen it was decided to remove this in a fast-followup PR.

Reply:
#1948